demos/CMakeLists.txt

```diff
@@ -35,3 +35,5 @@ create_demo(Demo_CSG "Demo CSG")
 create_demo(Demo_DoF "Demo DoF")
 create_demo(Demo_VR "Demo Virtual Reality")
 create_demo(Demo_Texturing "Demo Texturing")
+

```

Comment:
This file should not be changed, and as consequence should not be present in the pull request.
The easiest way to undo all the changes - is to _revert_ this file, for example, in GitHub Desktop application.

Reply:
I reverted the file to the last time when you modified